src/subtitles/subtitles.js

```diff
@@ -8,14 +8,20 @@ function Subtitles(mediaPlayer, autoStart, playbackElement, defaultStyleOpts, me
   let subtitlesContainer
 
   if (useLegacySubs) {
-    import("./legacysubtitles.js")
-      .then(({ default: LegacySubtitles }) => {
-        subtitlesContainer = LegacySubtitles(mediaPlayer, autoStart, playbackElement, mediaSources, defaultStyleOpts)
+    if (available()) {
+      import("./legacysubtitles.js")
+        .then(({ default: LegacySubtitles }) => {
+          subtitlesContainer = LegacySubtitles(mediaPlayer, autoStart, playbackElement, mediaSources, defaultStyleOpts)
+          callback(subtitlesEnabled)
+        })
+        .catch(() => {
+          Plugins.interface.onSubtitlesDynamicLoadError()
+        })
+    } else {
+      new Promise((resolve) => resolve()).then(() => {
         callback(subtitlesEnabled)
```

Comment:
Do we want to return subtitlesEnabled here or false? can't work out what it should be.

Reply:
I think you can toggle subtitles in the API regardless of if they're availible, so this would make the state be out of sync with it internally